### PR TITLE
Add new page feedback issue template

### PR DESCRIPTION
**Pull request by svrnm**
We discussed this a while back that with github's feature of YML-based form-based issue templates we could streamline the page feedback, PTAL

Preview of the form: https://github.com/svrnm/opentelemetry.io/blob/add-page-feedback-template-issue/.github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  if a page was helpful.
+title: '[Page Feedback]: Your feedback'
+body:
+  - type: input
+    id: url
+    attributes:
+      label: URL
+      description: >-
+        The URL of the website where you are facing an issue. If not
+        auto-populated please copy the URL from the browser address bar.
```
Comment (tiffany76) [inline .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml:13]:
```suggestion
        The URL of the webpage where you are facing an issue. If not
        auto-populated, please copy the URL from the browser address bar.
```

File: .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description:
+        'How has the page not been helpful? What information is missing? '
+      placeholder: >-
+        Provide details on the problem. The more information you share with us
+        the easier it is to help and to provide a solution
```
Comment (tiffany76) [inline .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml:25]:
```suggestion
        Provide details on the problem. The more information you share with us
        the faster we can resolve the issue.
```

**Review by tiffany76 — APPROVED** [review]
Couple small suggestions. I think we should emphasize that we want the page URL. Otherwise, LGTM.

File: .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+name: Page Feedback
+description: >-
+  Used for page feedback from users visiting opentelemetry.io and clicking "no"
+  if a page was helpful.
+title: '[Page Feedback]: Your feedback'
```
Comment (chalin) [inline .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml:5]: Is this the template title or the title that will be used when the template is instantiated? If the latter, can we replace "Your feedback" with "INSERT PAGE TITLE HERE" ... or just use "Page feedback". Or, add an optional field entitled "Page title".

**Review by chalin — APPROVED** [review]
Very nice. LGTM

Should you be linking to this template from the "no" button text in this PR too?

**Comment by svrnm** [comment]
> Very nice. LGTM
> 
> 
> 
> Should you be linking to this template from the "no" button text in this PR too?

I would do this in a separate PR. I had it as part of this PR initially but couldn't verify it working before it is merged.

File: .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml (hunk 3 above)
Comment (chalin) [reply .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml:5]: Sure.

**Review by theletterf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml (hunk 3 above)
Comment (svrnm) [reply .github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml:5]:
```suggestion
title: '[Page Feedback]: summary of your feedback'
```

**Pull request merged**